aspnetcore/fundamentals/middleware/index.md

```diff
@@ -298,9 +298,9 @@ The simplest possible ASP.NET Core app sets up a single request delegate that ha
 
 [!code-csharp[](index/snapshot/Middleware/Startup.cs)]
 
-The first <xref:Microsoft.AspNetCore.Builder.RunExtensions.Run*> delegate terminates the pipeline.
+The first <xref:Microsoft.AspNetCore.Builder.RunExtensions.Run%2A> delegate terminates the pipeline.
```

Comment:
Consider: In the previous section, original to the doc:  The introduction of a single request delegate pattern using a single anonymous function might benefit from a sub title for it.  Then the reader can go right to that pattern if that is what they are looking for.  Perhaps the "Chain multiple request delegates together with Use" section could also use a sub title for the same reason.

Reply:
Thanks for the suggestion. I'll open another PR to address that.